cellprofiler/gui/help.py

```diff
@@ -2027,287 +2027,323 @@
+            column shows 8 values for the slices collected in each stack, numbered from 0 to 7, followed by the slices
+            for other stacks.</li>
+        </ul>
+    </li>
+    <li>In the <b>NamesAndTypes</b> module, assign the channel to a name of your choice. If there are multiple
+    channels, you will need to do this for each channel. For this example, you could do the following:
+        <ul>
+            <li>Select "Assign images matching rules".</li>
+            <li>Make a new rule <code>[Metadata][Does][Have C matching][0]</code></li>
+            <li>Click the <img src="memory:{MODULE_ADD_BUTTON}">&nbsp;button to the right of the rule to add another
+            set of rules underneath.</li>
+            <li>Add the rule <code>[Image][Is][Stack frame]</code>. This combination tells CellProfiler not to treat
+            the image as a single file, but rather as a series of frames.</li>
+            <li>Name the image <i>DAPI</i>.</li>
+            <li>Click the "Add another image" button to define a second image with a set of rules.</li>
+            <li>Make a new rule <code>[Metadata][Does][Have C matching][1]</code></li>
+            <li>Click the <img src="memory:{MODULE_ADD_BUTTON}">&nbsp;button to the right of the rule to add another
+            set of rules underneath.</li>
+            <li>Add the rule <code>[Image][Is][Stack frame]</code>.</li>
+            <li>Name the image <i>GFP</i>.</li>
+            <li>Click the "Add another image" button to define a third image with a set of rules.</li>
+            <li>Make a new rule <code>[Metadata][Does][Have C matching][2]</code></li>
+            <li>Click the <img src="memory:{MODULE_ADD_BUTTON}">&nbsp;button to the right of the rule to add another
+            set of rules underneath.</li>
+            <li>Add the rule <code>[Image][Is][Stack frame]</code>.</li>
+            <li>Name the image <i>TxRed</i>.</li>
+            <li>In the "Image set matching method" setting, select "Metadata".</li>
+            <li>Select "FileLocation" for the DAPI, GFP and TxRed channels. The FileLocation metadata tag identifies
+            the individual stack, and selecting this parameter ensures that the channels are first matched within each
+            stack, rather than across stacks.</li>
+            <li>Click the <img src="memory:{MODULE_ADD_BUTTON}">&nbsp; button to the right to add another row, and
+            select <i>Series</i> for each channel.</li>
+            <li>Click the "Update" button below the divider to confirm that the proper image slices are listed and
+            matched across the channels. The corresponding <i>FileLocation</i> and <i>Series</i> for each channel
+            should now be matched to each other.</li>
+        </ul>
+    </li>
+    <li>In the <b>Groups</b> module, select the metadata that defines a distinct image stack. For the example above, do
```

Comment:
I'm not sure. I'm unfamiliar with this file format, and am not having much luck finding information about it on the internet.